examples/ble_uart_echo_client.py

```diff
@@ -0,0 +1,34 @@
+"""
+Used with ble_uart_echo_test.py. Transmits "echo" to the UARTService and receives it back.
+"""
+
+import time
+
+from adafruit_ble import BLERadio
+from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
+from adafruit_ble.services.nordic import UARTService
+
+ble = BLERadio()
+while True:
+    while ble.connected and any(UARTService in connection for connection in ble.connections):
```

Comment:
Yeah, we really need a helper for this. Maybe:

```
ble.find_service(UARTService)
```

With a behavior of `Helper method that looks through all connections for a given service and returns the first one found.`

Reply:
We're merging this now and will look for API helpers, etc. when rewriting the current examples. Thanks and we'll keep all these in mind.